src/unix/fs.c

```diff
@@ -229,9 +229,46 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 #endif
 }
 
+static ssize_t uv__fs_access(uv_fs_t * req) {
+#ifdef __MVS__
+  int r = 0;
+
+  if (req->flags | F_OK)
```

Comment:
On z/OS, if you are using F_OK to test for the file's existence, you cannot OR with any of the other symbols. So F_OK can only be tested by itself.